### PR TITLE
[RELEASE] v3.33.0

### PR DESCRIPTION
**Pull request by wasp898**
Release `v3.33.0`

### CODE REVIEW

**Review by TeslaAdis — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.33.0]
+
+### Added
+- Support for providing custom `aria-label` when ListBox is used.
```
Comment (HalfbyteHeroes) [inline CHANGELOG.md:10]:
```suggestion
- Support for providing custom `aria-label` when `ListBox` is used.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed
+- Updating the markers on live streams causing unhandled exception after player is destroyed
```
Comment (HalfbyteHeroes) [inline CHANGELOG.md:13]: For consistency with previous changelog entries.

```suggestion
- Updating the markers on live streams causing unhandled exception after player is destroyed.
```

**Review by mimcz — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -760,6 +768,7 @@ Version 2.0 of the UI framework is built for player 7.1. If absolutely necessary
 ## 1.0.0 (2017-02-03)
 - First release
 
+[3.33.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.32.0...v3.33.0
```
Comment (HalfbyteHeroes) [inline CHANGELOG.md:771]: Don't we usually also include a `develop` entry to compare the current release to `develop`?

File: CHANGELOG.md (hunk 2 above)
Comment (wasp898) [reply CHANGELOG.md:13]: Some have it, some don't. But sure :)

**Review by HalfbyteHeroes — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (wasp898) [reply CHANGELOG.md:771]: Yes, but not in this PR as we are merging to `main` here 👍

**Pull request merged**